EvalData/models/pairwise_assessment_document.py

```diff
@@ -4,6 +4,7 @@
 See LICENSE for usage details
 """
 # pylint: disable=C0103,C0330,no-member
+import json

```

Comment:
I wonder if we need to import the entire `json` package. Can we import only the specific functions we need, as in line 10?

Reply:
Fixed.